script/setup/install-cni

```diff
@@ -21,7 +21,19 @@
 #
 set -eu -o pipefail
 
-CNI_COMMIT=${1:-$(grep containernetworking/plugins "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')}
+GOMODEnv=`go env |grep 'GOMOD=' | awk -F '=' '{print $2}' | sed 's/\"//g'`
+NULLValue=/dev/null
+if [ -f "$GOPATH"/src/github.com/containerd/containerd/go.mod ];then
+    CNI_COMMIT=${1:-$(grep containernetworking/plugins "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')}
+elif [ "$NULLValue" != "$GOMODEnv" ];then
```

Comment:
When no go.mod, retuen `/dev/null` (on Unix-like systems), e.g. 
```shell
$go env GOMOD
/dev/null
```
Documentation (go help environment):
```
GOMOD
The absolute path to the go.mod of the main module.
If module-aware mode is enabled, but there is no go.mod, GOMOD will be
os.DevNull ("/dev/null" on Unix-like systems, "NUL" on Windows).
If module-aware mode is disabled, GOMOD will be the empty string.
```

Reply:
Sorry, this comment was written based on the previous code that used `go env | grep GOMOD ...` which returned an empty string rather than /dev/null for a missing GOMOD.

Reply:
Yes, if use `go env | grep GOMOD ...` can instead check for an empty string.